vidarrtest-regression.json.in

```diff
@@ -1,11 +1,11 @@
 [
   {
     "id": "TEST_0001_WG_HG38_FASTQ_INPUT",
-    "description": "runs the fingerprintsCollector workflow on a WG library using hg38, with fastq as input",
+    "description": "runs the hotspotFingerprintsCollector workflow on a WG library using hg38, with fastq as input",
```

Comment:
hotspotFingerprintCollector, not hotspotFingerprintsCollector
distinction is tha this workflow will generate a single fingerprint.  The dev workflow that it was based on was meant to generate multiple fingerprints

Reply:
done